assets/src/edit-story/components/moveable/moveStyle.js

```diff
@@ -42,38 +43,58 @@ export const GlobalStyle = createGlobalStyle`
 	.default-moveable .moveable-control.moveable-s,
 	.default-moveable .moveable-control.moveable-e,
 	.default-moveable .moveable-control.moveable-w {
-		border-radius: 8px;
+		border-radius: 4px;
+	}
+
+	.default-moveable .moveable-control.moveable-nw,
+	.default-moveable .moveable-control.moveable-ne,
+	.default-moveable .moveable-control.moveable-sw,
+	.default-moveable .moveable-control.moveable-se {
+	  border-radius: 0px;
```

Comment:
This should be 2px, no?
![Screen Shot 2021-02-08 at 17 13 26](https://user-images.githubusercontent.com/637548/107288209-10a32080-6a31-11eb-8415-574837f01374.png)